lib/src/Uncertainty/Distribution/KernelMixture.cxx

```diff
@@ -661,6 +661,55 @@ KernelMixture::PointWithDescriptionCollection KernelMixture::getParametersCollec
   return parameters;
 } // getParametersCollection
 
+
+Point KernelMixture::getParameter() const
+{
+  const UnsignedInteger dimension = getDimension();
+  const UnsignedInteger size = sample_.getSize();
+  Point parameter;
+  for (UnsignedInteger i = 0; i < dimension; ++ i)
+  {
+    for (UnsignedInteger j = 0; j < size; ++ j)
+    {
+      parameter.add(sample_[j][i]);
+    }
+    parameter.add(bandwidth_[i]);
+  }
+  return parameter;
+}
+
+Description KernelMixture::getParameterDescription() const
+{
+  const UnsignedInteger dimension = getDimension();
+  PointWithDescriptionCollection parametersCollection(getParametersCollection());
+  Description parameterDescription;
+  for (UnsignedInteger i = 0; i < dimension; ++i)
+  {
+    parameterDescription.add(parametersCollection[i].getDescription());
+  }
+  return parameterDescription;
```

Comment:
It would be much more efficient to do it the other way, move lines from getParametersCollection to set description, and call getParameterDescription from getParametersCollection.

Reply:
As I change the order I dont want to mix, so I rewrote getParameterDescription from scratch.

Reply:
I do not get it, getParametersCollection must call getParameterDescription, not the other way.

Reply:
I rewrote getParameterDescription to not call getParametersCollection.
getParametersCollection cannot call getParameterDescription as the order is different now because I changed the order of the i/j loops.